packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -1119,13 +1119,27 @@ export const transformers = (model: ModelData): string[] => {
 	if (model.pipeline_tag && LIBRARY_TASK_MAPPING.transformers?.includes(model.pipeline_tag)) {
 		const pipelineSnippet = ["# Use a pipeline as a high-level helper", "from transformers import pipeline", ""];
 
-		if (model.tags.includes("conversational") && model.config?.tokenizer_config?.chat_template) {
-			pipelineSnippet.push("messages = [", '    {"role": "user", "content": "Who are you?"},', "]");
+		if (model.config?.tokenizer_config?.chat_template) {
+			if (model.tags.includes("conversational")) {
```

Comment:
But image-text-to-text models are conversational too, no?

Reply:
yeah! I basically am a bit dense: https://github.com/huggingface/huggingface.js/pull/1434/commits/72e521231bf3a393adb25d3283dc00fd53469e89